go/pkg/cache/singleflightcache.go

```diff
@@ -34,6 +34,20 @@ func (s *SingleFlight) LoadOrStore(key interface{}, valFn func() (interface{}, e
 	return e.val, e.err
 }
 
+// Load is similar to a sync.Map.Load.
+func (s *SingleFlight) Load(key interface{}) (val interface{}, err error, loaded bool) {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+	var eUntyped interface{}
+	eUntyped, loaded = s.store.Load(key)
+	if loaded {
+		e := eUntyped.(*entry)
+		val = e.val
```

Comment:
Can you make the use of named return vars consistent in this file? They're used here but not in LoadOrStore

Reply:
yes, in a separate CL